bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableInstanceName.java

```diff
@@ -96,4 +102,29 @@ public String toTableNameStr(String tableId) {
   public BigtableTableName toTableName(String tableId) {
     return new BigtableTableName(toTableNameStr(tableId));
   }
+
+  /**
+   * Creates a fully qualified cluster within the current instance.
+   * @param clusterId the id of the cluster
+   * @return A fully qualified name for the cluster within the instance.
+   */
+  public String toClusterNameString(String clusterId) {
+    return instanceName + "/clusters/" + clusterId;
+  }
+  
+  /** @return the projectId */
+  public String getProjectId() {
+    return projectId;
+  }
+
+  /** @return the instanceId */
+  public String getInstanceId() {
+    return instanceId;
+  }
+
+  /** @return the fully qualified instanceName */
```

Comment:
From the ListClustersRequest docs:

>   // The unique name of the instance for which a list of clusters is requested.
>   // Values are of the form projects/&lt;project&gt;/instances/&lt;instance&gt;
>   // Use <instance> = '-' to list Clusters for all Instances in a project,
>   // for example "projects/myproject/instances/-"
>   string parent = 1;

There needs to be some good naming of instanceId which is part of the string vs. the entire fully qualified url structure.  If you have better wording for this special case of '-', I'm all ears.  alternatively, I could just use .toString() instead of getInstanceName()